### DISCUSSION TITLE
Unbelievable absence of records about Ringer's lactate in `inputevents` table

### DISCUSSION THREAD

**Discussion by ylee03** [MIT-LCP/mimic-code#1150] [MIMIC-IV]
### Prerequisites

* [X] Put an X between the brackets on this line if you have done all of the following:
    * Checked the online documentation: https://mimic.mit.edu/
    * Checked that your issue isn't already addressed: https://github.com/MIT-LCP/mimic-code/issues?utf8=%E2%9C%93&q=

### Description

Estimating osmolarity of whole infused intravenous fluids, I found a pure clinical issue in the MIMIC-IV (probably also in the MIMIC-III). Ringer's lactate solution is one of most commonly used iv fluid during fluid resuscitation. Indeed, one can easily obtain `itemids` from the `d_items` table, where 220953 (Ringers) and 220955 (Ringers lactate) are indicating it. But...

* There are no record about Ringer's lactate in the `inputevents` table.

`select *
from physionet-data.mimic_icu.inputevents
where itemid IN (220953, 220955);`

In my perspective as a clinician, its absence would make the dataset to be unreliable. If I made a mistake, please point it out.

Thanks,

Younsuk.

**Comment by SichengH** [comment]
LR in MIMIC-IV is under itemid 225828 and 225827. The two itemids that you mentioned are not in use, also mentioned in the d_items tables. Please let us know if you have any other questions about iv.
